utils/yolo/annotator.py

```diff
@@ -34,6 +35,7 @@ def __init__(self,
         self.model_object = self.client.get_model_version_by_id(
             model_version_id
         )
+        self.label_list = self.dataset_object.list_labels()
```

Comment:
J'aurais plutot transformé la liste de label en un dict {label_name: Label}, pour pouvoir trouver facilement dans le dict en fonction du nom du label et pas avoir a itérer sur la liste en entier